server/v2/stf/core_event_service.go

```diff
@@ -5,6 +5,7 @@ import (
 	"encoding/json"
 	"slices"
 
+	"github.com/cosmos/cosmos-sdk/codec"
```

Comment:
we shouldn't bring in a cosmos-sdk dependency, can we use directly gogoproto jsonpb?